### DISCUSSION TITLE
Follow link in doc using corfu-info-documentation

### DISCUSSION THREAD

**Discussion by minikN** [minad/corfu#410] [Ideas]
Hello,

When viewing documentation for the current candidate using `corfu-info-documentation`, sometimes the doc offers links to external documentation. Can I somehow open links like that? If I click on them with the mouse, the doc buffer as well as corfu just disappear.

If I summon a persistent buffer using `C-u M-h` this works of course, but I was wondering if it can be done not using a persistent buffer.

![image](https://github.com/minad/corfu/assets/1888338/4ec9dadd-c7b9-4264-9fde-64b703a27da8)

Thanks!

**Comment by minad** [comment]
No, this is not yet supported. We could convert the documentation buffer to a persistent buffer if a mouse click is detected inside the documentation window. The Corfu popup would still close since the documentation window will be selected after executing the mouse click. One could probably work around that by making sure that the original window will stay selected. If someone has interest in working on this, I am happy to review patches.
